lib/test2.exs

```diff
@@ -0,0 +1,9 @@
+defmodule PhoenixCourse.UserControllerTest do
+  @moduledoc false
+  use PhoenixCourse.ConnCase
+  alias PhoenixCourse.User
+
+  defp user_count do
+    Repo.one( from u in User, select: count(u.id) )
```

Comment:
There is no whitespace around parentheses/brackets most of the time, but here there is.